sqs/config.go

```diff
@@ -0,0 +1,155 @@
+package sqs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/sqs"
+	"github.com/aws/aws-sdk-go-v2/service/sqs/types"
+)
+
+type SubscriberConfig struct {
+	AWSConfig aws.Config
+
+	// How long about unsuccessful reconnecting next reconnect will occur.
+	ReconnectRetrySleep time.Duration
+	// Delete message attempts
+	CreateQueueInitializerConfig QueueConfigAttributes
+
+	GenerateCreateQueueInput GenerateCreateQueueInputFunc
+
+	GenerateGetQueueUrlInput GenerateGetQueueUrlInputFunc
+
+	GenerateReceiveMessageInput GenerateReceiveMessageInputFunc
+
+	GenerateDeleteMessageInput GenerateDeleteMessageInputFunc
```

Comment:
I've added config for all things where we can specify in AWS requests - all those has sane defaults

looking for feedback

I need to add comments here